package.json

```diff
@@ -11,8 +11,9 @@
     "truffle-compile": "truffle compile",
     "truffle-test": "yarn clean; yarn truffle-migrate && truffle test",
     "ganache": "ganache-cli -d",
-    "web:start": "cd ./web && npm start",
-    "web:test": "cd ./web && npm test",
+    "web:start": "cd ./web; yarn start",
+    "web:test": "cd ./web; yarn test",
+    "web:install": "cd ./web; yarn",
```

Comment:
Running `yarn` installs everything. We should just make sure this is apart of the setup steps

Reply:
Oh I guess this is for web. NVM makes sense.